app/src/test/java/org/mozilla/fenix/settings/deletebrowsingdata/DeleteAndQuitTest.kt

```diff
@@ -94,44 +102,52 @@ class DeleteAndQuitTest {
         }
     }
 
-    @Ignore("Intermittently failing; will be fixed with #5406.")
+    @Ignore("Failing test; need more investigation.")
```

Comment:
This one is still failing when I run it and remove this @ignore :/

Reply:
Yeah, I updated the test to follow current implementation and left a comment for now, but I don't have the bandwidth to debug all the tests.